pep-8011.rst

```diff
@@ -65,6 +65,18 @@ Roles and responsibilities of the leadership trio
   Just as Python looks at and learn from other communities for inspiration, other
   communities will look at Python and learn from us.
 
+
+Authority of the trio
+=====================
+
+To be clear, in case any dispute arises: the trio has the final
```

Comment:
Thanks for the PR. However I feel like this is also contradictory to the following text under "Formation of working group" heading. 

> Python Core developers will defer decisions to these working groups on their respective topic. However these groups will answer/defer to the trio.

Reply:
After re-reading the text and discussions, and pondering for a bit, here's another attempt to capture what you're going for:

```
To be clear, in case any dispute arises: the trio has the final
authority to pronounce on PEPs (except for the governance PEP), to
decide whether a particular decision requires a PEP, and to resolve
technical disputes in general. The trio's authority does not include
changing the governance itself, or other non-technical disputes that
may arise; these should be handled through the process described in
PEP 8001.
```

Is that closer?

Reply:
Yup, thanks so much! I've gone ahead and made the edit using this new text.